libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -127,9 +135,22 @@ export class MembersDynamicListComponent implements AfterViewInit, OnInit, OnCha
       this.groupId,
       this.selectedGroupStatuses
     );
+
+    // Checks if this is 'members' group
+    if (this.groupId) {
+      this.groupManager.getGroupById(this.groupId).subscribe(
```

Comment:
I am not completly sure for which case we need this info.

Reply:
This info is used as an indication, that the current group is 'members' group.